pkg/naming/names.go

```diff
@@ -100,6 +100,16 @@ func IndexFromName(n string) (int32, error) {
 	return int32(index), nil
 }
 
+func RackOrdinalFromName(name string, sc *scyllav1.ScyllaCluster) (int, error) {
+	for i := range sc.Spec.Datacenter.Racks {
+		if sc.Spec.Datacenter.Racks[i].Name == name {
+			return i, nil
+		}
+	}
+
+	return -1, fmt.Errorf("can't find rack %q in ScyllaCluster %q", name, ObjRef(sc))
```

Comment:
usually, when erroring out this is the default value like `nil, 0, ""` or at least I don't see the special value used

Reply:
removed func in favor of generic FindIndex